packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -472,6 +473,12 @@ class DropdownMenu<T> extends StatefulWidget {
   ///    and notifies its listeners on [TextEditingValue] changes.
   final List<TextInputFormatter>? inputFormatters;
 
+  /// The offset to adjust the alignment of the menu.
+  ///
+  /// This property is used to add spacing between the [MenuAnchor] and the [DropdownMenuEntry]s.
+  /// The default value is [Offset.zero], which means no additional spacing.
```

Comment:
[`MenuAnchor.alignmentOffset`](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/menu_anchor.dart#L176-L195) already has detailed and descriptive documentation; it'd be nice to include that here as well.

We can use dartdoc templates to avoid having 2 copies of the same documentation:

```diff
+ /// {@template flutter.material.MenuAnchor.alignmentOffset}
  /// The offset of the menu relative to the alignment origin determined by
  /// [MenuStyle.alignment] on the [style] attribute and the ambient
  /// [Directionality].
  ///
  /// Use this for adjustments of the menu placement.
  ///
  /// Increasing [Offset.dy] values of [alignmentOffset] move the menu position
  /// down.
  ///
  /// If the [MenuStyle.alignment] from [style] is not an [AlignmentDirectional]
  /// (e.g. [Alignment]), then increasing [Offset.dx] values of
  /// [alignmentOffset] move the menu position to the right.
  ///
  /// If the [MenuStyle.alignment] from [style] is an [AlignmentDirectional],
  /// then in a [TextDirection.ltr] [Directionality], increasing [Offset.dx]
  /// values of [alignmentOffset] move the menu position to the right. In a
  /// [TextDirection.rtl] directionality, increasing [Offset.dx] values of
  /// [alignmentOffset] move the menu position to the left.
  ///
  /// Defaults to [Offset.zero].
+ /// {@endtemplate}
  final Offset? alignmentOffset;
```

And then here, we'd just use

```suggestion
  /// {@macro flutter.material.MenuAnchor.alignmentOffset}
```

Then both members would show the same explanation when you hover your mouse in VS Code, and when you look it up on [api.flutter.dev](https://api.flutter.dev/flutter/material/MenuAnchor/alignmentOffset.html).

Reply:
I never knew this. Thank you for this because I think this would benefit our project at work as well!

Reply:
no prob :)